tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+    //TODO: This should move to app-platform for standardized typescript library builds
+    "compilerOptions": {
+        "skipLibCheck": true,
+        // Target latest version of ECMAScript.
+        "target": "esnext",
+        "declaration": true,
+        "module": "esnext",
+        // Search under node_modules for non-relative imports.
+        "moduleResolution": "node",
+        // Process & infer types from .js files.
+        // "allowJs": true,
+        // "checkJs": true,
+        // Enable strictest settings like strictNullChecks & noImplicitAny.
+        "strict": true,
+        // Disallow features that require cross-file information for emit.
+        // "isolatedModules": true,
+        // Import non-ES modules as default imports.
+        "esModuleInterop": true,
+        "jsx": "react",
+        "rootDir": "src"
+    },
+    // include entrypoint instead of wildcard to prevent adding
+    // untracked files
+    "include": ["src/App.tsx", "global.d.ts", "**/*.test.*"],
```

Comment:
Files will of course be included due to imports. However I find it annyoing if I have a file I've been working on, that is not imported anywhere, to cause errors when commiting. So I've opted to just specify the entrypoint here.